unittests/parser_test_case.py

```diff
@@ -84,3 +87,10 @@ def _test_calldef_exceptions(self, calldef, exceptions):
             (calldef.name,
              pprint.pformat([delc.name for delc in exception_decls]),
              pprint.pformat([delc.name for delc in exceptions_indeed])))
+
+if sys.version_info < (2, 7, 0) and not found_unittest2:
```

Comment:
I would prefer if we just raise an exception if unittest2 is not installed (that would be 2 lines in the try/except block). Then we can safely rely on the unittest2 library instead of having to monkeypatch stuff.

It's adding a dependency (only for testing) on python 2.6, but I think we can live with it (and I am personally not using python 2.6 anymore; the tests just need to work on Travis).